src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -169,6 +183,12 @@ public Response listItems(String categoryIdsParam, String locationIdsParam, Stri
 
   @Override
   public Response updateItem(UUID itemId, Item payload) throws Exception {
+    // TODO: Add resource based permission check
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=13&issues=AWQwvtHgbXXTsx7OQ1wW&open=AWQwvtHgbXXTsx7OQ1wW)

[ISSUE_KEY]: AWQwvtHgbXXTsx7OQ1wW